### PR TITLE
Added rake script to run some setup commands

### PR DESCRIPTION
**Pull request by ruthwickp**
The host application runs this task by calling `rake lightning`.

What are your thoughts on allowing arguments to the task (i.e. User) to set this up in code.

### CODE REVIEW

File: lib/tasks/lightning_tasks.rake [hunk 1]
```diff
@@ -1,4 +1,11 @@
...
+task :lightning do
+  # Task goes here
+  puts 'Copying migrations from engines...'
+  `bin/rails lightning:install:migrations`
+  puts 'Running engine migrations...'
+  `bin/rails db:migrate SCOPE=lightning`
+  puts 'Creating initializers script'
+  `echo "Lightning.flaggable_entities = []" > config/initializers/lightning.rb`
+  puts 'Successfully ran setup script'
+end
```
Comment (pranav7) [inline lib/tasks/lightning_tasks.rake]: The install script would be `rails lightning:install`

```suggestion
namespace :lightning do
  task :install do
    puts 'Copying migrations from engines...'
    `bin/rails lightning:install:migrations`

    puts 'Running engine migrations...'
    `bin/rails db:migrate SCOPE=lightning`

    puts 'Creating initializers script'
    `echo "Lightning.flaggable_entities = []" > config/initializers/lightning.rb`

    puts 'Successfully ran setup script'
  end
end
```

**Comment by pranav7** [comment]
> What are your thoughts on allowing arguments to the task (i.e. User) to set this up in code.

Yes, worth giving it a shot, would be fun! :)

**Comment by ruthwickp** [comment]
Updated rake script to produce the same outcome as the README instructions

**Comment by pranav7** [comment]
Nice one!! Looks good.

Did it run fine when you ran in the dummy app? Also, does it still work fine if we just run `rails lightning:install`?

**Comment by ruthwickp** [comment]
Yeah, I tried it on the dummy app and running `rake lightning:install User Workspace` worked fine.

**Review by pranav7 — APPROVED** [review]

**Pull request merged**